src/core/thread/topology.hpp

```diff
@@ -629,6 +686,10 @@ class Neighbor : public InstanceLocatorInit
 #endif
     uint8_t         mVersion;  ///< The MLE version
     LinkQualityInfo mLinkInfo; ///< Link quality info (contains average RSS, link margin and link quality)
+#if OPENTHREAD_CONFIG_MLE_LINK_METRICS_ENABLE
+    LinkedList<LinkMetricsSeriesInfo> mLinkMetricsSeriesInfoList; ///< A list of Link Metrics Forward Tracking Series
```

Comment:
When a neighbor is removed, the forward tracking series configured by it should be cleared?

Reply:
It should. Where are the calls should we clear it? I'll update it.

Reply:
Updated.